README.md

```diff
@@ -92,53 +92,80 @@ the swarm parameters for the test and the local url where the app is running
-     - And presumably other repo paths needed in the future for other environments
-5. Add secrets to your GitHub Actions.
-   - `AZURE_TENANT_ID` with the tenant ID from Azure Active Directory.
-   - `AZURE_SUBSCRIPTION_ID` with the ID from the subscription that everything should be deployed into.
-   - `AZURE_CLIENT_ID` with the ID of the App Registration created previously.
-
-#### Dev Environment Deployment
-
-The Dev environment is meant to be the Wild West.  Meaning anyone can push to it to test something, and there is no
-requirement that only good builds be pushed to it.  Use the Dev environment if you want to test something in a deployed
-environment.
-
-To deploy to the Dev environment...
+We have a number of environments that are split between CDC and non-CDC Azure Entra domains and subscriptions.
+
+##### Internal
+
+The Internal environment is meant to be the Wild West.  Meaning anyone can push to it to test something, and there is no
+requirement that only good builds be pushed to it.  Use the Internal environment if you want to test something in a
+deployed environment in a _non-CDC_ Azure Entra domain and subscription.
+
+To deploy to the Internal environment...
 1. Check with the team that no one is already using it.
-2. [Find the `dev` branch](https://github.com/CDCgov/trusted-intermediary/branches/all?query=dev) and delete it in
-   GitHub.
-3. Delete your local `dev` branch if needed.
+2. [Find the `internal` branch](https://github.com/CDCgov/trusted-intermediary/branches/all?query=internal) and delete
+   it inGitHub.
+3. Delete your local `internal` branch if needed.
    ```shell
-   git branch -D dev
+   git branch -D internal
    ```
-4. From the branch you want to test, create a new `dev` branch.
+4. From the branch you want to test, create a new `internal` branch.
    ```shell
-   git checkout -b dev
+   git checkout -b internal
    ```
 5. Push the branch to GitHub.
    ```shell
-   git push --set-upstream origin dev
+   git push --set-upstream origin internal
    ```
 
-Then the [deploy](https://github.com/CDCgov/trusted-intermediary/actions/workflows/dev-deploy.yml) will run.  Remember
-that you now have the `dev` branch checked out locally.  If you make subsequent code changes, you will make them on the `dev`
-branch instead of your original branch.
+Then the [deploy](https://github.com/CDCgov/trusted-intermediary/actions/workflows/internal-deploy.yml) will run.
+Remember that you now have the `internal` branch checked out locally.  If you make subsequent code changes, you will
+make them on the `internal` branch instead of your original branch.
+
+##### Dev
+
+The Dev environment is similar to the Internal environment but deploys to a CDC Azure Entra domain and subscription.  It
+is also meant to be the Wild West.  Dev deploys similarly to the Internal environment, but you interact with the
+`internal` branch.
+
+##### Staging
```

Comment:
Should we put a TDB on the staging environment until the new environment is created?  where is it deploying otherwise?  or... do we have 2 non-CDC environments now?

Reply:
We have two non-CDC environments: `internal` and `staging`. Soon we'll update this section to remove the `non-` part.

Reply:
And it's active today. It's been active since the beginning. The deletion I mentioned in Slack was just temporary. It's back in the same non-CDC Entrance domain as before.